monitoring/pra/alerts.yaml

```
@@ -0,0 +1,88 @@
+x-inputs:
+  - name: namespace
+    type: constant
+    value: zenko
+  - name: kafka_connect_src_job
+    type: constant
+    value: artesca-data-dr-source-base-queue-connector-metrics
+  - name: kafka_connect_sink_job
+    type: constant
+    value: artesca-data-dr-base-queue-connector-metrics
+  - name: dr_sink_instance
+    type: constant
+    value: artesca-data-dr
+  - name: rto_alert_threshold
+    type: config
+    value: 3600 # 1 hour
+  - name: mongo_jobs
+    type: constant
+    value: zenko/data-db-mongodb-sharded-shard.*
+  - name: lifecycle_jobs
+    type: constant
+    value: artesca-data-backbeat-lifecycle-.*-headless
+
+groups:
+- name: PraAlerts
+  rules:
+
+  - alert: DrResourcePausedWhileOtherRunning
+    expr: |
+      (sum(up{job="${kafka_connect_src_job}", namespace="${namespace}"}) or vector(0)) 
```

practically, we have the same value for both {kafka_connect_src_job} and {kafka_connect_snk_job} : so we also need to check drSinkInstance every time...

```suggestion
      (sum(up{drSinkInstancejob="", ${kafka_connect_src_job}", namespace="${namespace}"}) or vector(0))
      !=
      (sum(up{drSinkInstancejob="${dr_sink_instance}", job="${kafka_connect_sink_job}", namespace="${namespace}"}) or vector(0))
```

---

(everywhere...)

---

sorry, typo : `drSinkInstance` instead of `drSinkInstancejob`

---

done here : https://github.com/scality/Zenko/pull/2130/commits/1e37f0af0ca7f67b3912a02b25b5d07660e65ee9#diff-22adcb487a4c13646fe867dd2c3504bedf8e022c2183ff5fbd7b92448fd75038R30-R32
https://github.com/scality/Zenko/pull/2130/commits/1e37f0af0ca7f67b3912a02b25b5d07660e65ee9#diff-22adcb487a4c13646fe867dd2c3504bedf8e022c2183ff5fbd7b92448fd75038R42
https://github.com/scality/Zenko/pull/2130/commits/1e37f0af0ca7f67b3912a02b25b5d07660e65ee9#diff-22adcb487a4c13646fe867dd2c3504bedf8e022c2183ff5fbd7b92448fd75038R52-R54
https://github.com/scality/Zenko/pull/2130/commits/1e37f0af0ca7f67b3912a02b25b5d07660e65ee9#diff-22adcb487a4c13646fe867dd2c3504bedf8e022c2183ff5fbd7b92448fd75038R65-R67
https://github.com/scality/Zenko/pull/2130/commits/1e37f0af0ca7f67b3912a02b25b5d07660e65ee9#diff-22adcb487a4c13646fe867dd2c3504bedf8e022c2183ff5fbd7b92448fd75038R78-R82

---

done , in all expressions